### PR TITLE
Fixes for kinetic corrections

### PR DESCRIPTION
**Pull request by tbody-cfs**
@mwigram

Check https://github.com/cfs-energy/cfspopcon/compare/main...fixes_for_kinetic_corrections

### CODE REVIEW

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    target_electron_temp: Unitfull,
+    kappa_e0: Unitfull,
+    electron_mass: Unitfull = 1.0 * ureg.electron_mass,
+    electron_charge: Unitfull = 1.0 * ureg.elementary_charge,
+    SOL_conduction_fraction: Union[float, xr.DataArray] = 1.0,
+    flux_limit_factor_alpha: Union[float, xr.DataArray] = 0.15,
+) -> Union[float, xr.DataArray]:
+    """Factor to reduce the electron heat conduction (relative to Braginskii) due to kinetic effects, via a flux limiter.
+
+    #TODO: Needs reference and units need to be checked
```
Comment (tbody-cfs) [inline cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:97]: @mwigram

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+
+def calc_delta_electron_sheath_factor(
+    separatrix_electron_temp: Unitfull,
+    target_electron_temp: Unitfull,
+    SOL_momentum_loss_fraction: Union[float, xr.DataArray] = 0.0,
+) -> Union[float, xr.DataArray]:
+    """Offset to increase the sheath heat transmission factor (relative to fluid) due to kinetic effects, according to SOL-KiT.
+
+    #TODO: needs reference
```
Comment (tbody-cfs) [inline cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:133]: @mwigram

File: cfspopcon/formulas/scrape_off_layer/two_point_model/separatrix_SOL_collisionality.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    input_units=dict(separatrix_electron_density=ureg.m**-3, separatrix_electron_temp=ureg.eV, parallel_connection_length=ureg.m),
+)
+def calc_separatrix_SOL_collisionality(
+    separatrix_electron_density: Unitfull,
+    separatrix_electron_temp: Unitfull,
+    parallel_connection_length: Unitfull,
+) -> Unitfull:
+    """Calculate the separatrix SOL collisionality.
+
+    #TODO: needs reference
```
Comment (tbody-cfs) [inline cfspopcon/formulas/scrape_off_layer/two_point_model/separatrix_SOL_collisionality.py:17]: @mwigram

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+@wraps_ufunc(
+    return_units=dict(Spitzer_conduction_reduction_factor=ureg.dimensionless),
+    input_units=dict(
+        separatrix_electron_density=ureg.m**-3,
+        separatrix_electron_temp=ureg.eV,
+        parallel_connection_length=ureg.m,
+        target_electron_temp=ureg.eV,
+        kappa_e0=ureg.W / (ureg.eV**3.5 * ureg.m),
+        electron_mass=ureg.kg,  # TODO: electron mass in kg?
+        electron_charge=ureg.C,  # TODO: elementary charge in Coulomb?
```
Comment (tbody-cfs) [inline cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:79]: @mwigram

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py (hunk 1 above)
Comment (mwigram) [reply cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:97]: Reference: "The Plasma Boundary of Magnetic Fusion Devices", Peter C Stangeby, Chapter 26

Units are correct

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py (hunk 2 above)
Comment (mwigram) [reply cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:133]: D. Power et al, https://arxiv.org/pdf/2208.10862v1

File: cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py (hunk 4 above)
Comment (mwigram) [reply cfspopcon/formulas/scrape_off_layer/two_point_model/kinetic_corrections.py:79]: These are correct

File: cfspopcon/formulas/scrape_off_layer/two_point_model/separatrix_SOL_collisionality.py (hunk 3 above)
Comment (mwigram) [reply cfspopcon/formulas/scrape_off_layer/two_point_model/separatrix_SOL_collisionality.py:17]: Reference: "The Plasma Boundary of Magnetic Fusion Devices", Peter C Stangeby, Chapter 4

**Pull request closed without merging**